tools/ci_build/github/azure-pipelines/templates/component-governance-component-detection-steps.yml

```diff
@@ -18,15 +18,16 @@ steps:
           and(eq('${{parameters.condition}}', 'always'), always())),
           and(eq('${{parameters.condition}}', 'succeeded'), succeeded()))
     inputs:
-      # ignore unit tests in emscripten. emscripten unit tests are not used in onnxruntime build
-      # ignore onnx third_party directory. onnx third_party directory is not responsible for onnxruntime build
-      # because sometime there are multiple repo is checked out in the same pipeline, we also need to add the repo name, such as onnxruntime/ to the ignoreDirectories
-      # ignore onnxruntime-extensions directory. onnxruntime-extensions directory is not responsible for onnxruntime build
+      # ignore cmake/external these are not our responsibility
```

Comment:
It is not about who is responsible for it. It is about whether the files are actually used in our build or not. For example, onnx is a 3rd party dependency. It might have security vulnerabilities. If the version we are using has a known vulnerability, we should get warned by this scanner.